### PR TITLE
Russian translation for the "Code splitting with React.lazy and Suspense", "Avoid multiple page redirects", "Remove unused code", "Streams—The definitive guide", "Interact with NFC devices on Chrome for Android", "Color and contrast accessibility" articles.

### PR DESCRIPTION
**Pull request by klyok**
Russian translation to be autoupdated on the GitLocalize repo. Translated articles:

- Code splitting with React.lazy and Suspense ([english link](https://web.dev/code-splitting-suspense/))
- Avoid multiple page redirects ([english link](https://web.dev/redirects/))
- Remove unused code ([english link](https://web.dev/remove-unused-code/))
- Streams—The definitive guide ([english link](https://web.dev/streams/))
- Interact with NFC devices on Chrome for Android ([english link](https://web.dev/nfc/))
- Color and contrast accessibility ([english link](https://web.dev/color-and-contrast-accessibility/))

### CODE REVIEW

**Comment by jpmedley** [comment]
@klyok Thank you for the submission. I'm passing this to a colleague who handles our translation issues. Can you please sign the attached CLA. We can't merge this without it.

**Comment by klyok** [comment]
> @klyok Thank you for the submission. I'm passing this to a colleague who handles our translation issues. Can you please sign the attached CLA. We can't merge this without it.

The CLA is signed (@klyok, yEheKim@gmail.com, Dec 01, 2021 21:36 PST). Could you please do the rescan?

**Comment by jpmedley** [comment]
@devnook @MichaelSolati Is our CLA scanning broken?

**Comment by MichaelSolati** [comment]
@jpmedley it's not broken but it has been updated/changed (not by us). It looks like we need to manually rerun the CLA check.

**Comment by jpmedley** [comment]
Ho do I do that?

**Comment by MichaelSolati** [comment]
@jpmedley => https://threadit.app/thread/r6you1pnjkmbeya0usk2/message/awtxh4yo3nb5edegeatqsr0l

I re-ran it for this pr, but enjoy the vid for reference.

**Comment by chybie** [comment]
Getting a reviewer to scan it through.

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (sadym-chromium) [inline src/site/content/ru/fast/remove-unused-code/index.md:5]:
```suggestion
  Npm упрощает добавление кода в проект. Но так ли уж нужны все эти лишние байты?
```

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (sadym-chromium) [inline src/site/content/ru/fast/remove-unused-code/index.md:28]: This instruction is not translated.

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (sadym-chromium) [inline src/site/content/ru/fast/remove-unused-code/index.md:33]:
```suggestion
(Coverage) в DevTools также сообщает, какое количество CSS- и JS-кода в приложении
```

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (sadym-chromium) [inline src/site/content/ru/fast/remove-unused-code/index.md:40]: Keep the pattern **_"translated term" (original term)_**

```suggestion
Указав полную конфигурацию Lighthouse через Node CLI, с помощью аудита
«Неиспользуемый JavaScript» (Unused JavaScript) можно отслеживать, какое количество
неиспользуемого кода поставляется с приложением.
```

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (sadym-chromium) [inline src/site/content/ru/fast/remove-unused-code/index.md:46]: `Включите` can be misunderstood as "turn on" instead of "include". I'd use "добавить", which in turn can be misread as "add", but it less confusing for readers.

```suggestion
[Webpack Bundle Analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer). Добавьте этот плагин в файл
```

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (sadym-chromium) [inline src/site/content/ru/fast/remove-unused-code/index.md:64]:
```suggestion
При перезагрузке приложения с добавленным плагином отображается масштабируемая
```

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (sadym-chromium) [inline src/site/content/ru/fast/remove-unused-code/index.md:72]:
```suggestion
Эта визуализация позволяет определить, какие части пакета
больше других, а также получить представление обо всех импортируемых
библиотеках. Таким образом можно определить, есть ли у вас неиспользуемые
или ненужные библиотеки.
```

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (sadym-chromium) [inline src/site/content/ru/fast/remove-unused-code/index.md:78]:
```suggestion
На предыдущем изображении древовидной карты довольно много пакетов
в одном домене `@firebase`. Если вашему сайту нужен только компонент базы данных firebase,
то обновите настройки импорта и получите эту библиотеку:
```

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (sadym-chromium) [inline src/site/content/ru/fast/remove-unused-code/index.md:95]:
```suggestion
Если вы используете webpack, просмотрите [список плагинов, которые
```

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (sadym-chromium) [inline src/site/content/ru/fast/remove-unused-code/index.md:109]:
```suggestion
более «легкую» альтернативу. Однако также важно оценить сложность и необходимые
для этого усилия, прежде чем удалять библиотеку из приложения полностью.
```

**Review by sadym-chromium — CHANGES_REQUESTED** [review]
Few styling suggestions and misspellings.

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (klyok) [reply src/site/content/ru/fast/remove-unused-code/index.md:28]: It looks like it's a `{% code %}` part. "Instruction" doesn't appear on the [live page](https://web.dev/remove-unused-code/). What I see on the page is a list of instructions not included in the text of the article (they are included by the `{% Instruction … %}` code, I suppose):
image

Or am I understanding something wrong?

**Comment by OrKoN** [comment]
LGTM Code splitting with React.lazy and Suspense % suggestion

File: src/site/content/ru/lighthouse-performance/redirects/index.md
Comment (sadym-chromium) [inline src/site/content/ru/lighthouse-performance/redirects/index.md:3]:
```suggestion
title: Как избежать множественной переадресации страницы
```

File: src/site/content/ru/lighthouse-performance/redirects/index.md
Comment (sadym-chromium) [inline src/site/content/ru/lighthouse-performance/redirects/index.md:57]: We have to be consistent with original and translated terms. Either:
1. (I personally prefer) use translated and add original in parenthesis.
2. or use original term and add translation in parenthesis.
```suggestion
- [Исходный код для аудита **Избежание множественной переадресации страниц** (Avoid multiple page redirects)](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/redirects.js).
```

File: src/site/content/ru/lighthouse-performance/redirects/index.md
Comment (sadym-chromium) [inline src/site/content/ru/lighthouse-performance/redirects/index.md:59]: Same consistency here

**Comment by sadym-chromium** [comment]
"redirects" LGTM with small comments

File: src/site/content/ru/accessible/color-and-contrast-accessibility/index.md
Comment (sadym-chromium) [inline src/site/content/ru/accessible/color-and-contrast-accessibility/index.md:27]:
```suggestion
В [руководстве WebAIM](https://webaim.org/standards/wcag/) для текста рекомендуется минимальная контрастность уровня AA — 4,5:1.
```

File: src/site/content/ru/accessible/color-and-contrast-accessibility/index.md
Comment (sadym-chromium) [inline src/site/content/ru/accessible/color-and-contrast-accessibility/index.md:42]:
```suggestion
Для проверки цветового контраста можно использовать аудит «Специальные возможности» (Accessibility Audit) в Lighthouse.
```

**Comment by sadym-chromium** [comment]
color-and-contrast-accessibility LGTM

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (chybie) [reply src/site/content/ru/fast/remove-unused-code/index.md:28]: Did a quick check on the code. Created a feature request for that. #6982. Could you please help to provide the translation here in comment? I'll see if I have time to implement that or wait for the team to do it.

Comment (klyok) [reply src/site/content/ru/fast/remove-unused-code/index.md:28]: @chybie Sure. Here is the source I've compiled from the Instructions.js:

```
1. Press `Control+Shift+J` (or `Command+Option+J` on Mac) to open DevTools.
2. Click the **Network** tab.
3. Select the **Disable cache** checkbox.
4. Reload the page.
```

Here is my translation for this source:

```
1. Откройте DevTools, нажав `Control+Shift+J` (или `Command+Option+J`, если у вас Mac).
2. Перейдите на вкладку **Сеть** (Network).
3. Установите флажок **Отключить кеш** (Disable cache).
4. Перезагрузите страницу.
```

Comment (chybie) [reply src/site/content/ru/fast/remove-unused-code/index.md:28]: Thanks @klyok. Created a PR #6984  . Once the review approved, we can rebase and it will be translated.

**Comment by chybie** [comment]
Here are the reviewers for each post:

- Code splitting with React.lazy and Suspense [alexrudenko]
- Avoid multiple page redirects [sadym]
- Remove unused code (ru, en) [sadym]
- Color and contrast accessibility ([sadym]
- Streams—The definitive guide [maxkoretskyi]
- Interact with NFC devices on Chrome for Android [nechaev]

Once the all reviews are done, we can proceed to merge the PR.

File: src/site/content/ru/blog/nfc/index.md
Comment (vladimir-nechaev) [inline src/site/content/ru/blog/nfc/index.md:724]: пишется легко -> создается легко
Не уверен, что "Запись приложения Android" звучит достаточно понятно, но может я просто не знаком с терминологией.
Может "Ссылка на ..."? Как я понимаю это ведь указание на то, какое приложение нужно открыть для чтения метки.

File: src/site/content/ru/blog/nfc/index.md
Comment (vladimir-nechaev) [inline src/site/content/ru/blog/nfc/index.md:777]: Окажите поддержку

**Comment by klyok** [comment]
@chybie What should I need to do now for this PR to be accepted (so that translations will appear on GitLocalize)? Should I implement the proposed changes now?

**Comment by chybie** [comment]
Hi @klyok , sorry for late reply as I was OOO. Once you implemented the changes, then I will get reviewers to take a quick look. When it's all good, then we are good to go!

File: src/site/content/ru/blog/nfc/index.md
Comment (klyok) [reply src/site/content/ru/blog/nfc/index.md:543]: I think it's more applicable to the English source. Russian translation can then be updated based on the source changes.

File: src/site/content/ru/blog/nfc/index.md
Comment (klyok) [reply src/site/content/ru/blog/nfc/index.md:722]: Поправим на "Различать NDEF-записи легче всего, используя их `id`."

**Comment by klyok** [comment]
> Hi @klyok , sorry for late reply as I was OOO. Once you implemented the changes, then I will get reviewers to take a quick look. When it's all good, then we are good to go!

I think I'm done implementing the changes.

File: src/site/content/ru/accessible/color-and-contrast-accessibility/index.md
Comment (sadym-chromium) [reply src/site/content/ru/accessible/color-and-contrast-accessibility/index.md]: affirmative

File: src/site/content/ru/accessible/color-and-contrast-accessibility/index.md
Comment (sadym-chromium) [reply src/site/content/ru/accessible/color-and-contrast-accessibility/index.md]: affirmative

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (sadym-chromium) [reply src/site/content/ru/fast/remove-unused-code/index.md]: affirmative

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (sadym-chromium) [reply src/site/content/ru/fast/remove-unused-code/index.md]: affirmative

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (sadym-chromium) [reply src/site/content/ru/fast/remove-unused-code/index.md]: affirmative

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (sadym-chromium) [reply src/site/content/ru/fast/remove-unused-code/index.md]: affirmative

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (sadym-chromium) [reply src/site/content/ru/fast/remove-unused-code/index.md]: affirmative

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (sadym-chromium) [reply src/site/content/ru/fast/remove-unused-code/index.md]: affirmative

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (sadym-chromium) [reply src/site/content/ru/fast/remove-unused-code/index.md]: affirmative

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (sadym-chromium) [reply src/site/content/ru/fast/remove-unused-code/index.md]: affirmative

File: src/site/content/ru/fast/remove-unused-code/index.md
Comment (sadym-chromium) [reply src/site/content/ru/fast/remove-unused-code/index.md]: affirmative

File: src/site/content/ru/lighthouse-performance/redirects/index.md
Comment (sadym-chromium) [reply src/site/content/ru/lighthouse-performance/redirects/index.md]: affirmative

File: src/site/content/ru/lighthouse-performance/redirects/index.md
Comment (sadym-chromium) [reply src/site/content/ru/lighthouse-performance/redirects/index.md:52]: affirmative

File: src/site/content/ru/lighthouse-performance/redirects/index.md
Comment (sadym-chromium) [reply src/site/content/ru/lighthouse-performance/redirects/index.md]: affirmative

File: src/site/content/ru/lighthouse-performance/redirects/index.md
Comment (sadym-chromium) [reply src/site/content/ru/lighthouse-performance/redirects/index.md]: affirmative

**Review by sadym-chromium — APPROVED** [review]
* LGTM remove unused code.
* LGTM redirects.
* LGTM lighthouse.
* LGTM accessibility colors.

**Review by chybie — APPROVED** [review]

**Comment by chybie** [comment]
@jpmedley can you approve this? LGTM.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**
